src/game/common/thing/thingtemplate.cpp

```diff
@@ -474,9 +474,9 @@ void ThingTemplate::Init_For_LTA(const Utf8String &name)
     strncpy(buffer, name, ARRAY_SIZE(buffer));
     int i;
 
-    for (i = 0; buffer[i]; i++) {
+    for (i = 0; buffer[i]; ++i) {
```

Comment:
Why make this unrelated change?

Reply:
Decompiler shows like so. Plus Pre-increment is always the default that should be used unless post-increment is explicitly required.